src/style.css

```diff
@@ -1 +1,26 @@
 /* add styles here */
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
```

Comment:
The use of nested selectors like `.stars--1 .stars__star:nth-child(-n + 1)` is not supported in standard CSS. Consider refactoring these selectors to be flat and compliant with standard CSS, or use a CSS preprocessor like SASS.